spec/index.bs

```diff
@@ -7404,7 +7404,7 @@ typedef double GPUPipelineConstantValue; // May represent WGSL's bool, f32, i32,
 </div>
 
 <div algorithm>
-    <dfn abstract-op>validating shader binding</dfn>(binding, layout)
+    <dfn abstract-op>validating shader binding</dfn>(variable, layout)
```

Comment:
These vars can be linked - that would have caught this, too, as bikeshed errors when a var is used only once.
```suggestion
    <dfn abstract-op>validating shader binding</dfn>(|variable|, |layout|)
```